qcut/.agents/skills/pr-comments/scripts/export-all.sh

```
@@ -41,10 +41,9 @@ echo "--- Main Thread Comments ---"
 THREAD_TEMP=$(mktemp)
 trap "rm -f $THREAD_TEMP" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)